app/models/concerns/publishable.rb

```diff
@@ -3,6 +3,7 @@ module Publishable
 
   included do
     include ActiveModel::Validations
+    include PublishableQueries
```

Comment:
This redefines some scopes (published, not_published, visible, hidden) which are also defined inside ``Publishable``. Shouldn't the definitions be removed from ``Publishable``?

Reply:
Correct, I forgot to remove them.